__utils__/tsconfig/tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "removeComments": false,
     "sourceMap": true,
     "strict": true,
-    "target": "es2021"
+    "target": "es2022"
```

Comment:
Necessary to make `Object.hasOwn` available in TypeScript, but I'm not sure if Node.js 18 supports all features in `es2022`. The safe option would be to define `Object.hasOwn` in a file at `__typings__/global.d.ts`.